src/foam/nanos/geocode/GoogleMapsBoundary.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * @license
+ * Copyright 2017 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.nanos.geocode',
+  name: 'GoogleMapsBoundary',
+
```

Comment:
Add documentation:

Reply:
Added documentation to all of the classes